Corona-Warn-App/src/main/res/navigation/nav_graph.xml

```diff
@@ -272,8 +289,8 @@
         android:label="fragment_submission_tan"
         tools:layout="@layout/fragment_submission_tan">
         <action
-            android:id="@+id/action_submissionTanFragment_to_submissionResultFragment"
-            app:destination="@id/submissionResultFragment">
+            android:id="@+id/action_submissionTanFragment_to_submissionTestResultPendingFragment"
+            app:destination="@id/submissionTestResultPendingFragment">
```

Comment:
Here we should pop the stack to prevent back navigation

Reply:
```xml
        <action
            android:id="@+id/action_submissionTanFragment_to_submissionTestResultPendingFragment"
            app:destination="@id/submissionTestResultPendingFragment"
            app:popUpTo="@id/mainFragment"
            app:popUpToInclusive="false">
```
Right?

Reply:
Yes